themes/sensei-course-theme/theme.json

```diff
@@ -0,0 +1,3 @@
+{
+    "version": 2
+}
```

Comment:
Having this file makes it a 'block theme', but it's not triggering anything yet. Will need more work & research to properly set it up for FSE.